index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+    
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+<title>Maria's Portfolio</title>
+<link rel="stylesheet" href="styles.css">
+
+<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto+Slab:wght@300&display=swap" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto+Slab:wght@100;200&family=Tiro+Telugu&display=swap" rel="stylesheet">
+</head>
+<body>
+
+    <header>
+        <div class="top">
+            <label>Maria Hussain</label>
+            <nav>
+                <ul>
+                    <li>
+                        <img src="Images/Hamburger.png" alt="Menu"/>
+                    </li>
+                </ul>
+            </nav>
+        </div>
+    </header>
+
+    <section class="hero">
+        <div>
+            <div>
+           <h1 class="intro"> Hey There. I'm Maria </h1>
+           <h3>I am a Software Developer</h3>
+           <p class="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            do not hestiate to contact me.</p>
+            </div>
+        
+            <div >
+                <ul id="social-icons">
+                <li class="icon"><a href="https://github.com"><img src="Images/github.png" alt="Github"/></a></li>
+                <li class="icon"><a href="https://www.linkedin.com"><img src="Images/in.png" alt="linkedin"/></a></li>
+                <li class="icon"><a href="https://angel.com"><img src="Images/angel.png" alt="angel"/></a></li>
```

Comment:
Kindly check on this logo image it's not displaying on my browser;
![image](https://user-images.githubusercontent.com/61408860/170525488-3fbce51b-6350-403e-b946-6076114803af.png)

- [ ] Also I can see a some errors in the design kindly check the comparison below and fix it;

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/170526065-62a4d3fe-26aa-4c62-8ced-ca426f005868.png) |  <img width="205" alt="image" src="https://user-images.githubusercontent.com/61408860/170526196-e7836d90-0d4e-453a-aee9-446688ab49f3.png">